RSSDP/SsdpCommunicationsServer.cs

```diff
@@ -126,6 +126,10 @@ public void BeginListeningForBroadcasts()
                         {
                             _BroadcastListenSocket = ListenForBroadcastsAsync();
                         }
+                        catch (SocketException)
+                        {
+                            _logger.LogError("Failed to bind to port 1900. DLNA will be unavailable");
```

Comment:
Is this 1900 not available as a variable? I don't like how it's hardcoded in the message. Also I thought 1900 is for auto-discovery, not for DLNA?

Reply:
`ListenForBroadcastsAsync` only binds port 1900 (const).
Don't know if what *exactly* the port is used for, but it has something to do with DLNA

Reply:
http://www.upnp-hacks.org/upnp.html
universal plug & play, this would be the upnp port mapping, but could possibly be more